client/extensions/woocommerce/app/promotions/fields/README.md

```diff
@@ -0,0 +1,65 @@
+Form Fields
```

Comment:
These form fields components don't seem to be promotions specific. Would you consider moving them out from under promotions?

Reply:
I certainly would! I'd like to do that in a separate PR though, perhaps after the initial work on Promotions is done.